run_development_cycle.sh

```diff
@@ -14,9 +16,9 @@ clear
 
 # https://github.com/aantron/bisect_ppx/blob/master/doc/advanced.md#Jbuilder suggests
 # modifying the jbuild file for release. Let's modify it for tests instead.
-sed -i "s/^;\(.*bisect_ppx.*\)$/\1/" ocaml_autodoc/jbuild
-jbuilder build --dev ocaml_autodoc/ocaml_autodoc.exe
-sed -i "s/^\(.*bisect_ppx.*\)$/;\1/" ocaml_autodoc/jbuild
+sed -i "s/^;\(.*bisect_ppx.*\)$/\1/" ocaml_autodoc/dune
+dune build ocaml_autodoc/ocaml_autodoc.exe
```

Comment:
No more `--dev` with dune: profile is selected with option `--profile` and defaults to `dev`